check_process

```diff
@@ -15,8 +15,11 @@
 		upgrade=1        from_commit=72d530415c016b23b6ba62085272957b04d2cca6
 		backup_restore=1
 		multi_instance=1
-		port_already_use=0
 		change_url=1
 ;;; Options
 Email=
 Notification=none
+;;; Upgrade options
+	; commit=72d530415c016b23b6ba62085272957b04d2cca6
+		name=Name and date of the commit.
```

Comment:
Ideally would be nice to give a proper name to the commit :stuck_out_tongue_winking_eye: